base.rb

```diff
@@ -0,0 +1,48 @@
+require 'bundler'
+require 'yaml'
+require 'sinatra/base'
+require "sinatra/multi_route"
+require 'securerandom'
+require 'active_support/core_ext/hash'
+require 'active_support/inflector'
+require './lib/tendrl'
```

Comment:
The lib folder will be loaded only if it's in the ruby load path, which happens during the initialization. It's just a push to the environment variable. I don't know whether shotgun and puma load config.ru, but if they do, just add that line to config.ru and it'll work.